### PR TITLE
Connectors - Module 2: FY2020Q1 Quarterly refresh

### PR DESCRIPTION
**Pull request by andrewconnell**
Includes refresh from FY19Q4 #599 that wasn't merged in

### CODE REVIEW

File: ConnectorActionableMsgs/02 Cards and Actions/Demos/02-Send-Actionable-Messages/SendCardEmail/Program.cs [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+			authClient = PublicClientApplicationBuilder.Create(clientId)
+				.WithAuthority(AzureCloudInstance.AzurePublic, tenantId)
+				.WithRedirectUri("https://login.microsoftonline.com/common/oauth2/nativeclient")
+				.Build();
+
+      try
+      {
+        // Get the access token
+        var result = await authClient
+														.AcquireTokenInteractive(scopes)
```
Comment (dclaux) [inline ConnectorActionableMsgs/02 Cards and Actions/Demos/02-Send-Actionable-Messages/SendCardEmail/Program.cs]: Looks like you have an indentation issue here

File: ConnectorActionableMsgs/02 Cards and Actions/Demos/02-Send-Actionable-Messages/SendCardEmail/Program.cs [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      catch (Microsoft.Graph.ServiceException graphEx)
+      {
+        Output.WriteLine(Output.Error, "An exception occurred while making a Graph request.");
+        Output.WriteLine(Output.Error, "  Code: {0}; Message: {1}", graphEx.Error.Code, graphEx.Message);
+      }
+    }
+
+    static string LoadCardMessageBody(string cardType, string filepath)
+    {
+			string messageBody = string.Empty;
```
Comment (dclaux) [inline ConnectorActionableMsgs/02 Cards and Actions/Demos/02-Send-Actionable-Messages/SendCardEmail/Program.cs]: Indentation issue?

File: ConnectorActionableMsgs/02 Cards and Actions/LabFiles/SendCardEmail/Program.cs [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    static IPublicClientApplication authClient = null;
+    static readonly string[] scopes =
+    {
+      "User.Read", // Scope needed to read /Me from Graph (to get email address)
+      "Mail.Send"  // Scope needed to send mail as the user
+    };
+
+    static void Main(string[] args)
+    {
+			var valid = true;
```
Comment (dclaux) [inline ConnectorActionableMsgs/02 Cards and Actions/LabFiles/SendCardEmail/Program.cs]: Indentation

**Comment by andrewconnell** [comment]
@davidchesnut fixed the whitespace inconsistencies. I missed the mix of tabs & spaces last time, but standardized it this time. Sorry about that.

**Pull request merged**